src/main/java/org/jabref/logic/cleanup/MoveFilesCleanup.java

```diff
@@ -41,6 +44,8 @@ public List<FieldChange> cleanup(BibEntry entry) {
                 if (fileChanged) {
                     changed = true;
                 }
+            } catch (FileSystemException exception) {
+                fileSystemExceptions.add(exception);
```

Comment:
Why not adding all `IOException`? They are logged below.

Reply:
I made this change based on the previous PR (https://github.com/JabRef/jabref/pull/11484). I also think that it's better to add all of IOException. 

Thanks for the input.

Reply:
Think, we overlooked that thing in the PR (because of other comments 😅).

Side thing: Don't forget to run the OpenRewrite task and check checkstyle 😅